src/transformers/quantizers/quantizer_torchao.py

```diff
@@ -73,6 +73,50 @@ def validate_environment(self, *args, **kwargs):
                     )
                 else:
                     self.offload = True
+        if self.pre_quantized:
+            safe_globals = []
+            if self.quantization_config.quant_type == "int4_weight_only":
+                from torchao.dtypes.affine_quantized_tensor import (
+                    AffineQuantizedTensor,
+                    TensorCoreTiledAQTLayout,
+                    TensorCoreTiledLayoutType,
+                    ZeroPointDomain,
+                )
+
+                safe_globals += [
+                    AffineQuantizedTensor,
+                    TensorCoreTiledAQTLayout,
+                    TensorCoreTiledLayoutType,
+                    ZeroPointDomain,
+                ]
+            elif self.quantization_config.quant_type == "int8_weight_only":
+                from torchao.dtypes.affine_quantized_tensor import (
+                    AffineQuantizedTensor,
+                    PlainAQTLayout,
+                    PlainLayoutType,
+                    ZeroPointDomain,
+                )
+
+                safe_globals += [PlainAQTLayout, AffineQuantizedTensor, PlainLayoutType, ZeroPointDomain]
```

Comment:
just a nit! I think it would be nicer to have the same vertical layout as the other ones

Reply:
fixed !